pyp2rpm/archive.py

```diff
@@ -117,6 +133,8 @@ def extractor_cls(self):
 
         return file_cls
 
+
+
```

Comment:
I probably forgot to remove extra lines or I have added them accidentally wile editing previous functions, I will fix it.